File: C/Random/Rock Paper Scissors!/RPS.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
int main(){
    const int R=1;
    const int P=2;
    const int S=3;
    int user;
    int comp=rand()%(S-R+1)+R;
    printf("Rock: 1\n Paper: 2\n Scissors: 3\n pick a number between these two:");
    scanf("%d",&user);
    printf("i picked %d and you picked %d\n",comp,user);
    if(user==comp){
        printf("its a a tie, try again!");
        return 1;
    } else if(user==1 && comp==3) printf("Rock beats scissors! you win");
      else if(user==1 && comp==2) printf("Paper beats rock! Nah i'd Win");
      else if(user==2 && comp==1) printf("Paper beats rock! you win");
      else if(user==2 && comp==3) printf("Scissors beats paper! Nah i'd Win");
      else if(user==3 && comp==1) printf("Rock beats Scissors! Nah i'd Win");
      else if(user==3 && comp==2) printf("Scissors beats paper! you win");
}

File: Coding/Python/Rock Paper Scissors/RPS.py
import random
R=1
S=2
P=3
comp=random.choice([R,P,S])
user=int(input("Rock: 1\n Paper: 2\n Scissors: 3\n pick a number between these three:"))
print("i picked",comp)
print("and you picked",user)
if user==comp:
    print("its a a tie, try again!")
elif user==1 and comp==3:
    print("Rock beats scissors! you win")
elif user==1 and comp==2:
    print("Paper beats rock! Nah i'd Win")
elif user==2 and comp==1:
    print("Paper beats rock! you win")
elif user==2 and comp==3:
    print("Scissors beats paper! Nah i'd Win")
elif user==3 and comp==1:
    print("Rock beats Scissors! Nah i'd Win")
elif user==3 and comp==2:
    print("Scissors beats paper! you win")